src/link/Plan9.zig

```diff
@@ -172,6 +172,7 @@ fn putFn(self: *Plan9, decl: *Module.Decl, out: FnDeclOutput) !void {
         // each file gets a symbol
         fn_map_res.value_ptr.* = .{
             .sym_index = blk: {
+                try self.syms.append(gpa, undefined);
```

Comment:
Is this intentional that you append `undefined` twice in a row to the same container?

Reply:
Yes, they both are filled in with .z symbols later.
The first one has the name of the file, while the second one has a null name for pop off the stack.